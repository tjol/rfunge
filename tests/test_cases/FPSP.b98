"PSPF"4#v(1F2FD3FAQKP1F2FDEC:M:PT:PG:P1FNYPaF5FYL1F\D0"002."RSaaa**FMI|
        >a"deliaf gnidaol"dk,@                      @_v#-5IMF4R"1.25"0<
                                                      >0"1.32-"RVaF:MDXP@
                                                            @,k3"eq?"a<