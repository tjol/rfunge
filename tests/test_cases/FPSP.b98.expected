0.626382 0.750000 0.931596 0.750000 1.333333 1.259859 